dom.go

```diff
@@ -101,11 +101,16 @@ func (e *Element) Reconcile(oldComp Component) {
 			}
 		}
 
-		// TODO fix style reset
 		style := e.node.Get("style")
 		for name, value := range e.Style {
 			style.Call("setProperty", name, value)
 		}
+		for name := range oldElement.Style {
+			if _, ok := e.Style[name]; !ok {
+				// Style was removed.
```

Comment:
This comment seems unhelpful, I'd suggest removing it.

If you look at the greater context of this func, other similar sections (for properties, event listeners, etc.) do not have an equivalent comment. I don't think they need it either, at least not at this level. The actual single line of code below is equally descriptive IMO.